release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3621,3 +3621,30 @@ $ oc adm release info 4.11.43 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-44"]
+=== RHSA-2023:3915 - {product-title} 4.11.44 bug fixes and security update
+
+Issued: 2023-07-06
+
+{product-title} release 4.11.44, which includes security updates, is now available. This update includes a Red Hat security bulletin for customers who run {product-title} in FIPS mode. For more information, see link:https://access.redhat.com/security/vulnerabilities/RHSB-2023-001[RHSB-2023:001]. 
+
+Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:3915[RHSA-2023:3915] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:3914[RHSA-2023:3914] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.44 --pullspecs
+----
+
+[id="ocp-4-11-44-bug-fixes"]
+==== Bug fixes
+* Previously, because client TLS (mTLS) was configured on an ingress controller, and the certificate authority (CA) in the client CA bundle required more than 1MB of certificate revocation list (CRLs) to be downloaded, the CRL `ConfigMap` object could not be updated due to the size limitation on `ConfigMaps`. As a result of the missing CRLs, connections with valid client certificates may have been rejected with the error `unknown ca`. With this update, the CRL `ConfigMap` for each ingress controller no longer exists; instead, CRLs are downloaded directly by each router pod ensuring connections with valid client certificates are no longer rejected. (link:https://issues.redhat.com/browse/OCPBUGS-14456[*OCPBUGS-14456*])
```

Comment:
s/certificate revocation list/certificate revocation lists  ??
s/`ConfigMaps`/config maps  **_--  or --_**  `ConfigMap` object
s/may/could
s/CRL `ConfigMap`/CRL config maps **_--  or --_** CRL  `ConfigMap` object 
s/pod ensuring connections /pod, ensuring that connections   (Not 100% sure on the comma; I am a serial comma over-user)

Reply:
Changes made.